lib/sinon.js

```diff
@@ -12,14 +12,45 @@ var match = require("./sinon/match");
 
 module.exports = exports = require("./sinon/util/core");
 
+exports.assert = require("./sinon/assert");
+exports.collection = require("./sinon/collection");
 exports.extend = require("./sinon/extend");
 exports.match = match;
 exports.spy = require("./sinon/spy");
 exports.spyCall = require("./sinon/call");
 exports.stub = require("./sinon/stub");
+exports.mock = require("./sinon/mock");
+exports.expectation = require("./sinon/mock-expectation");
 exports.createStubInstance = require("./sinon/stub").createStubInstance;
 exports.typeOf = require("./sinon/typeOf");
 
+exports.log = function () {};
```

Comment:
sinon.logError uses it; I believe the idea is that consumers can supply their own `sinon.log` function to handle log messages raised by the fakeServer, fakeXhr, etc.  From the docs site:

> ### sinon.log(string)
> Logs internal errors, helpful for debugging. By default this property is a noop function, set it to something that prints warnings in your environment for more help, e.g. (if you are using JsTestDriver):
>
> ```sinon.log = function (message) {  jstestdriver.console.log(message); };```